addons/GodotInk/Src/InkStoryImporter.cs

```diff
@@ -51,15 +52,18 @@ public override Array<Dictionary> _GetImportOptions(string path, int presetIndex
 
     private static Error ImportFromInk(string sourceFile, string destFile, bool shouldCompress)
     {
-        using FileAccess file = FileAccess.Open(sourceFile, FileAccess.ModeFlags.Read);
+        using Godot.FileAccess file = Godot.FileAccess.Open(sourceFile, Godot.FileAccess.ModeFlags.Read);
 
         if (file == null)
-            return FileAccess.GetOpenError();
+            return Godot.FileAccess.GetOpenError();
 
         Compiler compiler = new(file.GetAsText(), new Compiler.Options
         {
             sourceFilename = sourceFile,
             errorHandler = InkCompilerErrorHandler,
+            fileHandler = 
+                new RelativePathFileHandler(
+                    Path.GetDirectoryName(sourceFile) ?? Directory.GetCurrentDirectory()),
```

Comment:
- `Path.GetDirectoryName(string)` should not be called on `sourceFile` here, rather on `file.GetPathAbsolute()`.  
- Falling back to `Directory.GetCurrentDirectory()` also feels weird. It should either fall back to `ProjectSettings.GlobalizePath("res://")`, or point-blank error out.

Reply:
Done. I originally had Directory.GetCurrentDirectory() to try to fall back to the existing logic, but I agree that ProjectSettings.GlobalizePath("res://") makes more sense for a Godot-specific handler.